controllers/configurationpolicy_controller.go

```diff
@@ -88,7 +88,10 @@ const (
 	reasonFoundNotApplicable = "Resource found but will not be handled in mustnothave mode"
 )
 
-var ErrPolicyInvalid = errors.New("the Policy is invalid")
+var (
+	ErrPolicyInvalid = errors.New("the Policy is invalid")
+	ErrMappingError  = errors.New("a mapping occurred")
```

Comment:
Is there a word missing here?
```suggestion
	ErrMappingError  = errors.New("a mapping error occurred")
```

Reply:
Oops. Fixed!